feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionDetailFragment.kt

```diff
@@ -68,12 +68,14 @@ class SessionDetailFragment : DaggerFragment() {
         binding.bottomAppBar.replaceMenu(R.menu.menu_session_detail_bottomappbar)
         binding.bottomAppBar.setOnMenuItemClickListener { item ->
             when (item.itemId) {
-                R.id.session_share ->
-                    Toast.makeText(
-                        requireContext(),
-                        "not implemented yet",
-                        Toast.LENGTH_SHORT
-                    ).show()
+                R.id.session_share -> {
+                    val session = binding.session ?: return@setOnMenuItemClickListener false
+                    @Suppress("StringFormatMatches") // FIXME
```

Comment:
> // FIXME

🤔 😹

Reply:
I'm sorry, i didn't understand. I fixed it 🙇